src/components/layout/progress/StepProgressBar/StepProgressBar.tsx

```diff
@@ -0,0 +1,155 @@
+import { useIsMobile } from '../../../../hooks';
+import {
+  AlertOctagonIcon,
+  AlertTriangleIcon,
+  CheckmarkIcon,
+} from '../../../icons';
+import './styles.css';
+
+function StepProgressBar({
```

Comment:
is their anything in `ant` we can use?

Reply:
actually yeah they do have one for this: https://ant.design/components/steps

But seems to be only customizable via className.... might have to do something like nth child for the individual styles on the steps. I really dislike that they dont provide an inline styles component.